data_steward/resource_files/fields/_deactivated_participants.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "type": "string",
```

Comment:
It might be better if this parameter is an `integer` instead of a `string`.  Especially since `person_id` is an `integer` in [omop tables](https://www.ohdsi.org/web/wiki/doku.php?id=documentation:cdm:person).